File: BACKEND/PickleballCourtBookingSystem/PickleballCourtBookingSystem.Core/Entities/CourtTimeSlot.cs
namespace PickleballCourtBookingSystem.Api.Models

{
    public class CourtTimeSlot
    {
        public Guid Id { get; set; }
        public DateTime Date { get; set; }
        public TimeSpan Time { get; set; }
        public int IsAvailable { get; set; }
        public double Price { get; set; }
        public Guid CourtId { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/maxholmes/androidapp/data/model/CourtTimeSlot.kt
package com.maxholmes.androidapp.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.time.LocalDate
import java.time.LocalTime

@Parcelize
data class CourtTimeSlot(
    var id: String,
    var date: String,
    var time: String,
    var isAvailable: Int,
    var price: Double,
    var courtId: String
) : Parcelable